gear-programs/historical-proxy/app/src/service.rs

```diff
@@ -0,0 +1,130 @@
+// Incorporate code generated based on the IDL file
+
+#[allow(dead_code)]
+pub(crate) mod vft {
+    include!(concat!(env!("OUT_DIR"), "/vft-manager.rs"));
+}
```

Comment:
proxy shouldn't depend on `vft-manager` in any form. I'd prefer dependency inversion here. Maybe even `sails` have ability to define some kind of interface.